google-cloud-tools-plugin/testSrc/com/google/gct/idea/appengine/cloud/AppEngineCloudConfigurableTest.java

```diff
@@ -155,7 +156,7 @@ private void initCloudConfigurable() {
   private File createTempFile() throws IOException {
     TemporaryFolder tempFolder = new TemporaryFolder();
     tempFolder.create();
-    File executable = new File(tempFolder.newFolder("bin"), "gcloud");
+    File executable = new File(tempFolder.newFolder("bin"), SystemInfo.isWindows ? "gcloud.cmd" : "gcloud");
```

Comment:
this code seems to be factored out in the CloudSdkUtilTest.  Also do we do this anywhere in the non-test code?